README.md

```diff
@@ -33,14 +30,15 @@ the project.
     - [Experimental Joint Matrix Support](#jm_support)
   - [Requirements](#requirements)
   - [Setup](#setup)
-    - [Compile with ComputeCpp](#compile-with-computecpp)
     - [Compile with DPC++](#compile-with-dpc)
     - [Compile with hipSYCL](#compile-with-hipsycl)
     - [Instaling portBLAS](#instaling-portBLAS)
-    - [POWER\_VR support (ComputeCpp Only)](#power_vr-support-computecpp-only)
     - [Doxygen](#doxygen)
     - [CMake options](#cmake-options)
-    - [Cross-Compile (ComputeCpp Only)](#cross-compile-computecpp-only)
+    - [ComputeCpp Compilation](#computecpp-deprecated)
```

Comment:
I think we can make this change here to make the title consistent with the one in the description of this section below.
```suggestion
    - [ComputeCpp Compilation (Deprecated)](#computecpp-deprecated)
```